src/components/button/_button.scss

```diff
@@ -142,6 +142,16 @@
     &.#{$prefix}--btn--ghost .#{$prefix}--btn__icon {
       margin: 0;
     }
+
+    &.#{$prefix}--tooltip--icon__top::after,
+    &.#{$prefix}--tooltip--icon__bottom::after {
+      transform: translate(calc(-50% + 8px), calc(100% + 10px));
+      margin: 0;
+    }
+  }
+
+  .#{$prefix}--btn--icon-only__wrapper.#{$prefix}--tooltip--icon {
+    display: inline-flex;
```

Comment:
What this is for?

Reply:
the `.bx--tooltip--icon` class has `display: flex`, but since this is a button we want it to be `display: inline-flex`, so I added the `.bx--btn--icon-only__wrapper.bx--tooltip--icon` class to increase the selector's specificity and change the display rule

Reply:
I see - Given I see icon tooltip tends to be used alongside with other contents, I think using `display: inline-flex` to `.bx--tooltip--icon` itself is safe enough.

Reply:
do you mean we should just make that change in `tooltip.scss` then?

Reply:
Right - BTW it's good to note (in `src/components/button/README.md`?) that the icon-only variant now requires tooltip style to be imported.

Reply:
done!